testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/model/BadRealmTest.java

```diff
@@ -33,6 +35,10 @@ public void testBadRealmName(KeycloakSession session) {
     @Test
     @ModelTest
     public void testBadRealmId(KeycloakSession session) {
+        if (Profile.isFeatureEnabled(Profile.Feature.MAP_STORAGE)) {
+            // when map storage is enabled, the id is always converted into a valid UUID.
+            return;
+        }
```

Comment:
```suggestion
        Assume.assumeFalse("When map storage is enabled, the id is always converted into a valid UUID.",  Profile.isFeatureEnabled(Profile.Feature.MAP_STORAGE));
```

Reply:
I've tried this suggestion and the test fails when the assumptions turns out to be false. With newer versions of Junit we have stuff like `@DisabledIf` annotations that can be used to disable the test under some circumstances, but the version we have in Keycloak is too old for that.

Reply:
Leaving this piece as is - assumeFalse is throwing exception and failing the test instead of skipping it.